kafka-elastic/src/main/java/com/baeldung/kafka/configs/ImperativeElasticsearchConfig.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.kafka.configs;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.elasticsearch.client.ClientConfiguration;
+import org.springframework.data.elasticsearch.client.elc.ElasticsearchClients;
+import org.springframework.data.elasticsearch.client.elc.ElasticsearchConfiguration;
+import org.springframework.data.elasticsearch.support.HttpHeaders;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Configuration
+public class ImperativeElasticsearchConfig extends ElasticsearchConfiguration {
+
+    private String elasticsearchUri = "elasticsearch:9200";
```

Comment:
If this is a constant use private static final and capital letters

Reply:
since its used only in one place, lets remove the variable completely